kaldb/src/main/java/com/slack/kaldb/logstore/search/LogIndexSearcher.java

```diff
@@ -5,5 +5,5 @@
 
 public interface LogIndexSearcher<T> extends Closeable {
   SearchResult<T> search(
-      String dataset, String query, long minTime, long maxTime, int howMany, AggBuilder aggBuilder);
+      String dataset, String query, Long minTime, Long maxTime, int howMany, AggBuilder aggBuilder);
```

Comment:
Should these two variables also be called startTimeMsEpoch/endTimeMsEpoch?

Reply:
As discussed we'll leave these as-is for now.